setup.py

```diff
@@ -5,7 +5,7 @@
     description='Script to parse DICOM files from a vNav series and convert them into motion scores',
     url='https://github.com/MRIMotionCorrection/parse_vNav_Motion',
     author='Dylan Tisdall',
-    author_email='mtisdall@mail.med.upenn.edu',
+    author_email='mtisdall@pennmedicine.upenn.edu',
     license='MIT',
     packages=['vnav'],
     install_requires=['pydicom', 'numpy'])
```

Comment:
Now that `setup.py` handles the dependencies, The readme can be updated to something like:

```
## Dependencies

The script depends on the pydicom and numpy packages.  These will be installed automatically when `pip install` is run
```